### DISCUSSION TITLE
Revisão de Exercícios da Trilha de Python 2024

### DISCUSSION THREAD

**Discussion by endersonmenezes** [codaqui/institucional#191] [Trilhas de Ensino]
Postem os novos exercícios e temas que acham interessante a gente abordar na nova versão do certificado.

**Comment by endersonmenezes** [comment]
**Capitulo 5**: Estrutura de Repetições

```python
numbers = [1,2,3,4,5]
for number in numbers:
  if number == 3:
    break
print(number)
```
- Qual a saída do console se executarmos o código acima?

**Comment by Testecodaqui** [comment]
Tarefa 2
Acesse a sua foto de perfil  e clique na área "your repositories"
Clique em "new"
Digite o nome de seu usuário, (.)ponto, github, (.)ponto e io
Que vai ficar deste jeito:"Seuusuário.github.io"
Clique em "Add a README file"
Depois clique em "Create repository"
Com este arquivo o repositório irá se tornar um site

https://github.com/codaqui/institucional/assets/164950179/e04fbb4e-50e0-40e0-9ac0-37210f4eae9d


Tarefa 3
Você já aprendeu a criar um repositório(tarefa 2), porém desta vez vc irá criar um repositório com seu nome
Voce ira insirir em "repository name" o nome de seu usuário apenas
Isso fara seu perfil do Github único, pois voce pode complementar com imagens e emojis, apenas deixe sua criatividade tornar realidade
Para gerenciar seu perfil README clique aqui "docs.github.com/pt/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme"

https://github.com/codaqui/institucional/assets/164950179/518f3f29-0d05-4e8a-9276-983d33bf5030

**Comment by LucaoCode** [comment]
**Exercicios de Estrutura Condicional**

**Exercicio 1:** Crie um programa que leia um número inteiro e mostre na tela se ele é PAR ou ÍMPAR:

**Resposta:**
```
n = int(input('Escolha um numero: ')) #Variavel que pedirá ao usuário um numero inteiro

if n%2==0: # inicio da estrutura condicional para verificar SE o numero é divisivel por 2

  print('Esse numero é PAR') # Mostra na tela que o numero é PAR

else: # Final da estrutura condicional para caso a primeira validação não der verdadeiro

  print('Esse numero é Ímpar') # Mostra na tela que o numero é Ímpar
```

**Exercicio 2:** #Escreva um programa que leia a velocidade de um carro. Se ele ultrapassar 80Km/h, mostre uma mensagem dizendo que ele foi multado. A multa vai custar R$7,00 por cada Km acima do limite.

**Resposta:**

```
km = float(input('Digite a velocidade do carro: ')) #Variavel que pedirá ao usuário um numero flutuante

if km >= 80.0: # Inicio de uma estrutura condicional

  calc = (km - 80)*7.0 #realiza o calculo solicitado pela questão

  print('Você ultrapassou o limite com {} terá que pagar R${}'.format(km,calc)) #exibe na tela o resultado caso maior ou igual a 80

else:

  print('você está dentro do limite permitido com {}'.format(km)) #exibe na tela o resultado caso menor que 80
```
